mirrord-layer/src/file/hooks.rs

```diff
@@ -41,7 +41,13 @@ unsafe fn open_logic(raw_path: *const c_char, open_flags: c_int) -> RawFd {
     if IGNORE_FILES.is_match(path.to_str().unwrap_or_default()) || !path.is_absolute() {
         FN_OPEN(raw_path, open_flags)
     } else {
-        let open_options = OpenOptionsInternalExt::from_flags(open_flags);
+        let open_options: OpenOptionsInternal = OpenOptionsInternalExt::from_flags(open_flags);
+        let read_only = ENABLED_FILE_RO_OPS
+            .get()
+            .expect("Should be set during initialization!");
+        if *read_only && !open_options.is_read_only() {
+            return FN_OPEN(raw_path, open_flags);
```

Comment:
I think we should somehow warn the user that the process writing to a file on his local machine and not the remote host